requirements.txt

```diff
@@ -1,3 +1,4 @@
 numpy>=1.16.0
 pandas>=0.25.0
 pvlib>=0.7.0
+scipy>=1.4.0
```

Comment:
is scipy>=1.4 required for the functions being used here?

Reply:
`scipy.stats.median_absolute_deviation` was added in 1.3. I can reduce the requirement to that.